flake.nix

```
@@ -7,6 +7,10 @@
     empty.url = "github:steveej/empty";
     empty.flake = false;
 
+    # workaround to getting passing the `.git` directory into the release-automation tests
```

getting passing?

---

```suggestion
    # workaround to allow the passing in of the `.git` directory into the release-automation tests
```

is this understandable?

---

Yes, this is good